SPECS/cassandra-cpp-driver/cassandra-cpp-driver.spec

```diff
@@ -0,0 +1,59 @@
+Summary:        DataStax C/C++ Driver for Apache Cassandra and DataStax Products
+Name:           cassandra-cpp-driver
+Version:        2.16.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/datastax/cpp-driver
+Source0:        https://github.com/datastax/cpp-driver/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  libuv-devel
+BuildRequires:  openssl-devel
+Requires:       libuv
+
+%description
+A modern, feature-rich, and highly tunable C/C++ client library for Apache
+Cassandra and DataStax Products using Cassandra's native protocol and Cassandra
+Query Language along with extensions for DataStax Products.
+
+%package devel
+Summary:        Development libraries for ${name}
+Group:          Development/Tools
+Requires:       %{name} = 1:%{version}-%{release}
```

Comment:
Please remove the `1:` epoch in this requires statement- since the base packages does not have an `Epoch` set, this will cause the `devel` subpackage to be uninstallable.